src/index.ts

```diff
@@ -94,39 +96,43 @@ const config = Config.fromFile("./config.toml");
 
   const shoot = async () => {
     const swaps = await comitNode.getSwaps();
-    log(`Found ${swaps.length} swap(s)`);
+    logger.trace(`Found ${swaps.length} swap(s)`);
 
     for (const swap of swaps) {
       const id = swap.properties ? swap.properties.id : undefined;
       try {
         const selectedAction = await actionSelector.selectActions(swap);
         if (selectedAction) {
-          log(
+          logger.trace(
             `Selected action for swap ${id}: ${JSON.stringify(selectedAction)}`
           );
           const executionResult = await actionExecutor.execute(
             selectedAction,
             config.maxRetries
           );
-          log(
+          logger.trace(
             `Action execution response for swap ${id}: ${JSON.stringify(
               executionResult
             )}`
           );
         } else {
-          log(`No action returned for swap ${id}`);
+          logger.trace(`No action returned for swap ${id}`);
         }
       } catch (err) {
-        log(
+        logger.debug(
```

Comment:
🔧I think this should be error... it even says `error has occurred`
```suggestion
        logger.error(
```